tests/unit/components/reception/SharedTestsForImportAndScanIn.js

```diff
@@ -0,0 +1,166 @@
+// Shared tests for the functionality of importing and scanning labware into LabWhere,
+// available on the Reception page
+//
+import * as Reception from '@/services/traction/Reception.js'
+import { expect, it } from 'vitest'
+import { store } from '@support/testHelper.js'
+
+export const sharedTestsForImportAndScanIn = (
+  wrapper,
+  scanBarcodesInLabwhereLocation,
+  mockShowAlert,
+) => {
+  describe('Unsuccessful Import ', () => {
+    vi.mock('@/services/labwhere/client.js')
+    vi.spyOn(console, 'error').mockImplementation(() => {})
+
+    describe('Fails to find labware', () => {
+      beforeEach(async () => {
+        wrapper.vm.reception.fetchFunction.mockRejectedValue('Failed fetch')
+        await wrapper.vm.fetchLabware()
+        wrapper.vm.reception.fetchFunction = vi.fn()
+        vi.spyOn(Reception, 'createReceptionResource').mockImplementation(() => {})
+      })
+      it('shows error alert message', () => {
+        expect(mockShowAlert).toHaveBeenCalledWith('Failed fetch', 'danger')
+      })
+
+      it('does not call create resource functionality', async () => {
```

Comment:
Again you are just testing something has not been called. You should really only test the output which looks like it is already being tested.